docs/sources/review/doc-validator/errata.md

```diff
@@ -1,7 +1,10 @@
 ---
 title: Errata for doc-validator
-menuTitle: Errata
+menuTitle: Errata for doc-validator
```

Comment:
I'm not in love with using the term "errata" here since it does not match the dictionary definition.  And assuming that people make the leap from "It's Latin for error, so those must be error messages" is a bit much.  Could we just call this "Doc-validator error messages"?

Reply:
The page is automatically generated, so we can't make chagnes here. We have to make it in the script. Along with the aliases have to be added to the output.